packages/customize-widgets/src/components/sidebar-block-editor/sidebar-adapter.js

```diff
@@ -135,7 +135,7 @@ export default class SidebarAdapter {
 		};
 	}
 
-	addWidget( widget ) {
+	addWidget( widget, index ) {
```

Comment:
Maybe default this to something e.g. `index = 0`?

Reply:
IMHO, this method should remain private, thus we don't need to assign defaults to make it easy to use. Instead, we should mark `index` as required explicitly to prevent us from introducing unexpected bugs 😅 .